src/core/server/app/authenticators/oidc/nonce.ts

```diff
@@ -0,0 +1,55 @@
+import cookie from "cookie";
+import crypto from "crypto";
+import { CookieOptions, Request, Response } from "express";
+import compare from "tsscmp";
+
+const COOKIE_NAME = "oidc:nonce";
+const COOKIE_OPTIONS: CookieOptions = {
+  httpOnly: true,
+  secure: true,
```

Comment:
We don't use https in development, this would prevent the cookie from being set, at least this is so in Chrome.